broker-daemon/utils/logger.js

```diff
@@ -55,10 +55,7 @@ function createLogger () {
       filterSensitive(),
       winston.format.timestamp(),
       winston.format.json()
-    ),
-    json: true,
```

Comment:
are these parameters not real?

Reply:
I think it was for an old version of winston